src/diffusers/schedulers/scheduling_repaint.py

```diff
@@ -190,6 +190,7 @@ def set_timesteps(
 
         timesteps = np.array(timesteps) * (self.config.num_train_timesteps // self.num_inference_steps)
         self.timesteps = torch.from_numpy(timesteps).to(device)
+        self.timesteps += self.config.steps_offset
```

Comment:
repaint scheduler wasn't doing this but other schedulers do, I assume this step is supposed to be here? (it doesn't seem to affect output much)